src/Tizen.Uix.Tts/Tizen.Uix.Tts/TtsClient.cs

```diff
@@ -658,6 +705,37 @@ public Mode CurrentMode
             }
         }
 
+        /// <summary>
+        /// Gets the current status of screen reader.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        /// <value>
+        /// The current status of screen reader.
+        /// </value>
+        /// <returns>
+        /// Boolean value whether screen reader is on or off.
+        /// </returns>
+        /// <feature>
+        /// http://tizen.org/feature/speech.synthesis
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
+        /// <exception cref="NotSupportedException">This exception can be due to TTS not supported.</exception>
+        public bool IsScreenReaderOn
+        {
+            get
+            {
+                bool isOn = true;
+                TtsError error = TtsCheckScreenReaderOn(_handle, out isOn);
+                if (error != TtsError.None)
+                {
+                    Log.Error(LogTag, "Get Mode Failed with error " + error);
```

Comment:
I think it will be better to change the error message like "Failed to check screen reader on with error"

Reply:
Thank you for checking it.
I miss the log text. I will fix it soon.